feature/guia nuevas practicas (#1525)
mercadopago/devsite-docs: guides/best-practices/ux-for-checkouts/ease-of-conversion.pt.md

# Facilidade de conversão

> Para que o processo de pagamento seja prático e rápido para seus usuários e, assim, obter mais conversões, seu checkout deve ser fácil de navegar e consistente. Siga estas dicas para alcançá-lo.

Reviewer: ```suggestion
Para que o processo de pagamento seja prático e rápido para seus usuários e assim obter mais conversões, seu checkout deve ser fácil de navegar e consistente. Siga estas dicas para alcançá-lo.
```

## Forneça etapas claras

Aconselhamos que você construa uma experiência com várias etapas onde cada uma delas se concentre em uma ação diferente, principalmente se for uma experiência mobile. Desta forma, o usuário pode focar sua atenção e é mais fácil encontrar o que procura em um piscar de olhos.

> Para **dispositivos móveis**, opte por etapas curtas, que chamem a atenção e sejam fáceis de navegar. No caso do **desktop**, por outro lado, você pode usar experiências "one step", já que o usuário consegue controlar melhor suas ações, ficando mais fácil encontrar o que busca.

Reviewer: ```suggestion
Para **dispositivos móveis**, opte por etapas curtas, que chamem a atenção do usuário e permitam uma navegação mais amigável. No caso do **desktop**, você pode usar experiências "one step", já que o usuário consegue controlar melhor suas ações, ficando mais fácil encontrar o que busca.
```

![pt Checkout agil](/images/best-practices-guide/PortCreaUnChoAgilIntro.png)

## Ofereça consistência 

> A **consistência** é um fator essencial para gerar coerência e congruência. Nos ajuda a estabelecer padrões que os usuários entendam e lembrem. Tente manter a consistência visual e textual.

Reviewer: ```suggestion
A **consistência** é um fator essencial para gerar coerência e congruência e ajuda a estabelecer padrões que os usuários entendam e lembrem. Tente manter a consistência visual e textual.
```

![pt Consistencia](/images/best-practices-guide/PortCreaUnChoAgilConsistenciaDoDonts.png)

> Agora, você conta com todas as informações de que precisa para criar a melhor experiência de pagamento para seus clientes. Integre nossos produtos e otimize a experiência do usuário com essas boas práticas.

Reviewer: ```suggestion
Agora, você conta com todas as informações necessárias para criar a melhor experiência de pagamento para seus clientes. Integre nossos produtos e otimize a experiência do usuário com essas boas práticas.
```

> PREV_STEP_CARD_PT
>
> Otimize sua experiência de checkout
>

> > Confira os componentes principais de um checkout e saiba como otimizá-los para que seus usuários tenham uma experiência ainda melhor, gerando mais conversão.

Reviewer: ```suggestion
> Conheça os componentes principais de um checkout e saiba como oferecer a melhor experiência de compra aos usuários.
```
